MapboxCoreNavigation/NavigationRouteOptions.swift

```diff
@@ -23,31 +23,32 @@ open class NavigationRouteOptions: RouteOptions {
         includesSteps = true
         routeShapeResolution = .full
         attributeOptions = [.congestionLevel, .expectedTravelTime]
-        includesExitRoundaboutManeuver = true
+        includesVoiceInstructions = true
```

Comment:
This will become `includesSpokenInstructions` in mapbox/MapboxDirections.swift#191.